webrtc/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Added support for insecure/deprecated signature verification algorithms, opt in via `SettingsEngine::allow_insecure_verification_algorithm` [#342](https://github.com/webrtc-rs/webrtc/pull/342).
 * Make RTCRtpCodecCapability::payloader_for_codec public API [#349](https://github.com/webrtc-rs/webrtc/pull/349).
 * Fixed a panic in `calculate_rtt_ms` [#350](https://github.com/webrtc-rs/webrtc/pull/350).
+* Change log level for packet reception in mDNS [#366](https://github.com/webrtc-rs/webrtc/pull/366).
```

Comment:
This should be specific for the `mdns` crate, so in `mdns/CHANGELOG.md`. Sorry I was clearer about that.